frontend/src/App.jsx

```
@@ -0,0 +1,209 @@
+                        return true;
+                    }
+
+                    // Metod 2: Fallback - Kolla på namnet (eftersom vi vet att petName finns i myRecords)
+                    // Vi jämför namnen och gör dem små bokstäver för att vara säkra
+                    if (r.petName && selectedPet?.name &&
+                        r.petName.toLowerCase() === selectedPet.name.toLowerCase()) {
+                        return true;
+                    }
+
+                    return false;
+                });
```

Biome (2.4.11)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)